docs/v1.0/out_forward.txt

```diff
@@ -336,6 +336,22 @@ The client private key path for TLS.
 
 The client private key passphrase for TLS.
 
+### keepalive
+
+| type | default | version |
+|:----:|:-------:|:-------:|
+| bool | fales    | 0.14.3  |
```

Comment:
We should specify concrete version such as `1.4.3` in version column.

Reply:
Thank you for your comment. As current fluentd looks v1.4.2(https://github.com/fluent/fluentd/releases/tag/v1.4.2). I changed the version to 1.4.3.